### PR TITLE
docs(tutorials): content validation - MTA-1594

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: tutorials/deploy-nextcloud-s3/index.mdx [hunk 1]
```diff
@@ -31,15 +31,15 @@ Combining Nextcloud with Object Storage gives you infinite storage space for you
...
+- You have a compute instance running on Ubuntu Focal Fossa (20.04)
 - You have [installed and configured](/tutorials/mariadb-ubuntu-bionic/) MariaDB on your Virtual Instance
 - You have sudo privileges or access to the root user.
 
 </Message>
 
 ## Installing Nextcloud
 
-1. Connect yourself to your server via SSH.
+1. Login to your server via SSH.
```
Comment (RoRoJ) [inline tutorials/deploy-nextcloud-s3/index.mdx:42]:
```suggestion
1. Log in to your server via SSH.
```

File: tutorials/setup-mysql/index.mdx [hunk 2]
```diff
@@ -64,21 +36,21 @@ Oracle provides a `.deb` package to configure the MySQL repository on your machi
...
 
 2. Install the package using `dpkg` by running the following command:
 
   ```
-  dpkg -i mysql-apt-config_0.8.15-1_all.deb
+  dpkg -i mysql-apt-config_0.8.20-1_all.deb
   ```
 
-3. During installation, you are prompted which MySQL product you want to install. You can keep the default configuration. Confirm by pressing OK:
+3. During installation, you are prompted which MySQL product you want to install. You can keep the default configuration (MySQL 8 Server). Confirm by pressing OK.
```
Comment (RoRoJ) [inline tutorials/setup-mysql/index.mdx:48]: Step should start with action verb

File: tutorials/setup-mysql/index.mdx [hunk 3]
```diff
@@ -64,21 +36,21 @@ Oracle provides a `.deb` package to configure the MySQL repository on your machi
...
 
 4. The tool will generate a configuration file for the `apt` package manager. You can verify its content by running the command `cat /etc/apt/sources.list.d/mysql.list`. The output should look like the following example:
```
Comment (RoRoJ) [inline tutorials/setup-mysql/index.mdx:50]: Step should start with action verb

File: tutorials/setup-mysql/index.mdx [hunk 4]
```diff
@@ -64,21 +36,21 @@ Oracle provides a `.deb` package to configure the MySQL repository on your machi
...
   ```
-  dpkg -i mysql-apt-config_0.8.15-1_all.deb
+  dpkg -i mysql-apt-config_0.8.20-1_all.deb
   ```
 
-3. During installation, you are prompted which MySQL product you want to install. You can keep the default configuration. Confirm by pressing OK:
+3. Select **MySQL 8 server** when you are prompted which MySQL product you want to install. Confirm by pressing OK.
 
-4. The tool will generate a configuration file for the `apt` package manager. You can verify its content by running the command `cat /etc/apt/sources.list.d/mysql.list`. The output should look like the following example:
+4. Verify the configutration file for the `apt` package manager, generated during installation of the package above, by running the command `cat /etc/apt/sources.list.d/mysql.list`. The output should look like the following example:
```
Comment (ofranc) [inline tutorials/setup-mysql/index.mdx:50]:
```suggestion
4. Verify the configuration file for the `apt` package manager, generated during the installation of the package above, by running the command `cat /etc/apt/sources.list.d/mysql.list`. The output should look like the following example:
```

File: tutorials/setup-mysql/index.mdx [hunk 5]
```diff
@@ -64,21 +36,21 @@ Oracle provides a `.deb` package to configure the MySQL repository on your machi
...
 
 2. Install the package using `dpkg` by running the following command:
 
   ```
-  dpkg -i mysql-apt-config_0.8.15-1_all.deb
+  dpkg -i mysql-apt-config_0.8.20-1_all.deb
   ```
 
-3. During installation, you are prompted which MySQL product you want to install. You can keep the default configuration. Confirm by pressing OK:
+3. Select **MySQL 8 server** when you are prompted which MySQL product you want to install. Confirm by pressing OK.
```
Comment (ofranc) [inline tutorials/setup-mysql/index.mdx:48]:
```suggestion
3. Select **MySQL 8 server** when you are prompted which MySQL product you want to install. Click **OK** to confirm.
```

**Review by ofranc — APPROVED** [review]

File: tutorials/deploy-nextcloud-s3/index.mdx [hunk 6]
```diff
@@ -31,15 +31,15 @@ Combining Nextcloud with Object Storage gives you infinite storage space for you
 
 - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
 - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
-- You have a compute instance running on Ubuntu Bionic Beaver (18.04)
+- You have a compute instance running on Ubuntu Focal Fossa (20.04)
```
Comment (ofranc) [inline tutorials/deploy-nextcloud-s3/index.mdx:34]:
```suggestion
- You have an Instance running on Ubuntu Focal Fossa (20.04)
```

**Review by ofranc — APPROVED** [review]

**Pull request merged**